CMakeLists.txt

```diff
@@ -481,7 +481,7 @@ if (LLAMA_CUDA)
         if (LLAMA_CUDA_F16 OR LLAMA_CUDA_DMMV_F16)
             set(CMAKE_CUDA_ARCHITECTURES "60;61;70") # needed for f16 CUDA intrinsics
         else()
-            set(CMAKE_CUDA_ARCHITECTURES "52;61;70") # lowest CUDA 12 standard + lowest for integer intrinsics
+            set(CMAKE_CUDA_ARCHITECTURES "86") # lowest CUDA 12 standard + lowest for integer intrinsics
```

Comment:
This shouldn't be changed. I imagine it was set for faster builds while testing, you can achieve the same by configuring with `-DCMAKE_CUDA_ARCHITECTURES=86`.

Reply:
My apologies, that wasn't supposed to be there. It's part of a different problem where I can't run the CUDA code anymore if I don't do that. I plan to open an issue about that.